.github/workflows/coverage-report.yaml

```diff
@@ -0,0 +1,73 @@
+name: Code coverage report to Codecov
+
+on:
+  push:
+    branches: 
+      - master
+jobs:
+        
+  coverage:
+    runs-on: ubuntu-latest
+        
+    name: Create and upload coverage
+    steps:
+        
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 8
+            
+      - name: Checkout code           
+        uses: actions/checkout@v2
+        
+      - name: Install Necessary Packages
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y net-tools libssl-dev gdb
+          sudo apt install gcovr
```

Comment:
why not at the above line using `apt-get`?

Reply:
No specific reason actually, when we realized gcovr should be installed and the suggested installation command was 'apt install gcovr' in container. It was put as it is. Making it `apt-get`